### DISCUSSION TITLE
Can't get to work. What am I missing?

### DISCUSSION THREAD

**Discussion by awey14** [slyh/Cities2-TrafficLightsEnhancement#52] [Q&A]
I was previously using this mod with Bep 6.0 and it was working perfectly, but decided to go to 5.0 due to the availability of other mods. I deleted Bep 6.0 and the version of this mod that is related to 6.0 and installed 5.0, as well as correlating version of the mod. However, I can't seem to get this mod to work. The prompt is reading that both the CommonLibraries and TrafficLightsEnhancement are loaded, but it does not show up when I go to the option in game. Did I miss getting rid of a file when I was converting over to 5.0 that may be conflicting and causing the issue? 

Other mods are working with Bep 5.0 such as SchoolCapacityBalancer and Cim Behavior Improvements. 

I have even tried using the Thunderstore app with the Alpha version of this mod. 

I am looking for any ideas I may had not tried yet. I have even uninstalled and reinstalled Cities 2 just in case the issue was somewhere in there.

**Comment by JimPacella** [comment]
Someone will probably be able to help you if you make it very clear about what directories you unziped files to and if you registered the mod in the Thunderstore app.

**Reply by awey14** [reply]
Sure. I always follow the directions given. 

First, I followed the directions for Bep and unzipped those files into the Cities 2 directory and the mods went into the plugins folder within the BepInEx folder. 
(C:)>Program Files(x86)>Steam>steamapps>common>Cities Skyline II>BepInEx>plugins

I did start the game before adding the mods into the plugins folder as the directions stated to create the plugins folder. 

For Thunderstore app: I downloaded and installed the app through Overwolf. I then opened Thunderstore mod manager and installed the apps through there. I would run Cities by pressing the Modded button in Thunderstore. All the mods would work as far as I could tell except for the TrafficLightsEnhancement. 

I am not sure what you mean by register the mod in the Thunderstore app.

**Reply by JimPacella** [reply]
When you launch the game through Thunderstorm, it does some magic of moving files around so that the mods in the profile you're launching are find-able by the game.  So if the traffic mod isn't in those folders too, it might not be loaded by the game when launched from thunderstorm.  

But I could be wrong, I only used the mod manager for a week.

**Reply by JimPacella** [reply]
Near the very end of this video, Biffa explains how to make the mod work with thunderstore https://www.youtube.com/watch?v=OUGg1QQiZLU

**Reply by awey14** [reply]
I had found those directions before and followed them with no luck.  I just followed them again with no luck.  I am attaching some screenshots in case it may help.  
![ThunderStore](https://github.com/slyh/Cities2-Various-Mods/assets/152909711/bb852def-4666-4d7e-9046-58b0f6ff4dde)
![InGameBep](https://github.com/slyh/Cities2-Various-Mods/assets/152909711/96f04c89-418d-4bb2-bf9f-b21c1a6c9777)
![InGame](https://github.com/slyh/Cities2-Various-Mods/assets/152909711/99503428-6d19-4235-a922-76e45b34fd26)


Thank you for taking the time to help.  I appreciate it.

**Reply by brewcrew87** [reply]
Your last screenshot shows the Traffic Lights Enhancement mod in the upper left hand corner...  You just need to select a junction.  What am I missing here?  You are in underground mode when you took the screenshot too.

**Reply by awey14** [reply]
Sigh, this is what I was missing.  I could have sworn the menu option to choose what kind of intersection was opened before placing the stop lights at the intersection was shown before hitting the intersection.  
Either way, thank you for your time and input.  I am truly grateful for the extra set of eyes.
